CMakeLists.txt

```diff
@@ -114,6 +114,12 @@ function(spvtools_default_compile_options TARGET)
     endif()
   endif()
 
+  if (MSVC)
+    # Specify /EHsc for exception handling. This makes using SPIRV-Tools as
+    # dependencies in other projects easier.
+    target_compile_options(${TARGET} PRIVATE /EHsc)
```

Comment:
You are right. I haven't thought carefully about the indications of adding `/EHsc` flag. After reading the [spec](https://msdn.microsoft.com/en-us/library/1deeycx5.aspx) a little bit more, I think `/EHs` is more suitable for SPIRV-Tools?